kong.conf.default

```diff
@@ -185,6 +194,15 @@
                                  # You can use the `kong hybrid` command to
                                  # generate the certificate/key pair.
 
+#cluster_ca_cert =               # The trusted CA certificate file in PEM
+                                 # format used to verify the `cluster_cert`.
+                                 # Required if `cluster_mtls` is set to `pki`.
+
+#cluster_server_name =           # The Server Name used in the DP/CP TLS
```

Comment:
"The server name used when DP TLS handshakes with CP. Sent to CP as SNI server name extension value as well as validating the CP cert returned. Must match the Common Name (CN) or Subject Alternative Name (SAN) inside the CP certificate."

or something like this should explain the use case for the server name property better.